src/journey-maps/angular/services/map/map-journey-service.ts

```diff
@@ -33,7 +33,7 @@ export class SbbMapJourneyService {
       const properties = feature.properties!;
       const type = properties.type;
       const pathType = properties.pathType;
-      const legId = properties.legId ?? 'journey'; // default: all belong together
+      const legId = properties.legId ?? 'trip'; // default: all belong together
```

Comment:
With these changes the method & thus the class name could be changed aswell, right?

Reply:
In principle, yes. On the other hand, we still have a TransferService and a RouteService class, so maybe better to just leave them all for now for consistency and then rename them when we delete v1 / TransferService ?

Reply:
On another note, I'm not very happy with this magic string 'journey' / 'trip'. [wtf](https://www.osnews.com/story/19266/wtfsm/). I realise it's not your code, but maybe we could figure out a cleaner way to do this ? Ideally we should not store a string into a variable which is meant to contain integers. Or at the very least use a constant and make sure the same constant is referred to everywhere the code checks for this value... as is, I can't even review this code change because I don't know what it might break.